From 0ddbafe9f90c587858a8259ac6a34c66e8a55345 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 2 Aug 2024 14:40:46 +0200
Subject: [PATCH 1114/1695] drivers: rstctrl: fix function description

Fix the description of functions rstctrl_dt_get_func() and
rstctrl_register_provider().

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git aacb34f18e476f63ca17b2b1a51f906cc015f29d drivers: rstctrl: fix function description]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ic0eb7b83125b6c50153e597d83522967ac616159
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/396601
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/include/drivers/rstctrl.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/include/drivers/rstctrl.h b/core/include/drivers/rstctrl.h
index d1112404d..8fff2933f 100644
--- a/core/include/drivers/rstctrl.h
+++ b/core/include/drivers/rstctrl.h
@@ -203,14 +203,14 @@ TEE_Result rstctrl_dt_get_by_name(const void *fdt, int nodeoffset,
  * devicetree properties
  *
  * @args: Pointer to devicetree description of the reset controller to parse
- * @data: Pointer to data given at rstctrl_dt_register_provider() call
+ * @data: Pointer to data given at rstctrl_register_provider() call
  * @rstctrl: Output reset controller reference upon success
  */
 typedef TEE_Result (*rstctrl_dt_get_func)(struct dt_pargs *args, void *data,
 					  struct rstctrl **out_rstctrl);
 
 /**
- * rstctrl_dt_register_provider - Register a reset controller provider
+ * rstctrl_register_provider - Register a reset controller provider
  *
  * @fdt: Device tree to work on
  * @nodeoffset: Node offset of the reset controller
-- 
2.47.2

